framework/assets/yii.js

```diff
@@ -146,7 +146,7 @@ yii = (function ($) {
         handleAction: function ($e, event) {
             var $form = $e.attr('data-form') ? $('#' + $e.attr('data-form')) : $e.closest('form'),
                 method = !$e.data('method') && $form ? $form.attr('method') : $e.data('method'),
-                action = $e.attr('href'),
+                action = $e.attr('href') || $e.data('href'),
```

Comment:
Ok, I'll try to do some other fixes with preserving href and solve problem of this bug